theta_topic_name: /theta_s/image_raw
